lib/pages/resetpassword/reset_password_page.dart

```diff
@@ -20,6 +30,23 @@ class _ResetPasswordPageState extends ConsumerState<ResetPasswordPage> {
 
   @override
   Widget build(BuildContext context) {
+    ref.listen<ResetPasswordState>(resetPasswordViewModelProvider, (_, state) {
+      state.maybeWhen(
+        error: (error) {
+          ScaffoldMessenger.of(context).showSnackBar(SnackBar(
+              content: Text(error ?? context.localization.loginError)));
```

Comment:
hmm, login error? 🤔 I think it's better to reuse a general error message, or you can define your own reset password failed message rather than using wrong error message here 😅

Reply:
@manh-t Good spot, as we have `error` param, in case of null, I will fallback to `errorGeneric` instead.